### PR TITLE
Mergebranch

### PR DESCRIPTION
**Pull request by ricardocarva**
Merging code with what we've done in our pair programming

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -18,10 +18,21 @@ jobs:
           command: sudo apt-get install -y npm
       - run:
           name: Install Dependencies
-          command: npm install --save-dev -y jest
+          command: |
```
Comment (ricardocarva) [inline .circleci/config.yml:21]: Added all dependencies trying to fix precision issue. It didnt, but I fixed the code though.

File: public/assets/images/mortgage.png
Comment (ricardocarva) [inline public/assets/images/mortgage.png:1]: Love the images

**Review by ricardocarva — COMMENTED** [review]
lgtm

**Review by ricardocarva — COMMENTED** [review]
54/54 files viewed

**Review by UFCarlosMartinez — APPROVED** [review]
Approve

**Pull request merged**